src/azure-cli/azure/cli/command_modules/appconfig/feature.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-# pylint: disable=line-too-long, too-many-statements, too-many-locals, too-many-branches
+# pylint: disable=line-too-long, too-many-statements, too-many-locals, too-many-branches, broad-except
```

Comment:
Is `broad-except` still needed?

Reply:
My bad. This is not needed anymore. Thanks for pointing this out.